.gitignore

```diff
@@ -0,0 +1 @@
+*.zwc
```

Comment:
Hey, I'd suggest you use a global ignore file (`git config --global core.excludesfile`) to ignore things that are specific to your installation :)

Reply:
OK. I'll drop this commit and keep the typo fix one.